apis/core/bootstrap/entrypoint.ts

```diff
@@ -0,0 +1,9 @@
+import { turtle } from '@tpluscode/rdf-string'
+import { hydra } from '@tpluscode/rdf-ns-builders'
+
+export const entrypoint = turtle`
+<> {
+  <> a ${hydra.Resource} ;
```

Comment:
Hydra crash course 💥

`hydra:Resource` is defined as 

> a class of dereferencable resources

Anything which has this class and exists in its own named graph will be immediately available as resource on the API surface (via `labyrinth`)